templates/_common/bootstrap.hierarchy/features.html

```diff
@@ -113,6 +113,16 @@ <h4 class="panel-title">
                                     <span class="text">
                         <span class="keyword highlight"><%= step.keyword %></span>
                           <span> <%- step.name %></span>
+
+                          <% if(step.arguments && step.arguments.length) { %>
```

Comment:
@asiermarques please un-commit this change as _bootstrap_ theme already prints arguments [here](https://github.com/gkushang/cucumber-html-reporter/blob/develop/templates/_common/bootstrap.hierarchy/features.html#L126)